File: src/main/java/no/fintlabs/linkwalker/Application.java
package no.fintlabs.linkwalker;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableAsync
@ComponentScan("no.fintlabs")
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: src/main/kotlin/no/fintlabs/linkwalker/Application.kt
package no.fintlabs.linkwalker

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class FintCoreLinkWalkerApplication

fun main(args: Array<String>) {
	runApplication<FintCoreLinkWalkerApplication>(*args)
}
